### PR TITLE
Fix #220 - mcm variable `:set` method is called twice

### LINKED ISSUE #220: MCM. setting variable in dropdown menu
```
	page.sidebar:createDropdown{ label = "Change the Weather", options = weatherList, inGameOnly = true,
	variable = mwse.mcm.createVariable{
		get = function(self) return tes3.getCurrentWeather().index or 0; end,
		set = function(self, newVal) tes3.getWorldController().weatherController:switchImmediate(newVal);
	 end}};
```
Setter function will be called twice.

### PR DESCRIPTION
**Pull request by C3pa**
The `self.variable.value` was updated twice: in `Dropdown:selectOption` and `Dropdown:update`. I decided to drop the `Dropdown:update` method and directly call `Parent.update(self)` in `Dropdown:selectOption`

Fixes #220.

### CODE REVIEW

**Comment by C3pa** [comment]
Here is an example that triggers the issue:
```lua

local function registerModConfig()
	local template = mwse.mcm.createTemplate({
		name = "Test Mod",
	})
	template:register()
	local page = template:createSideBarPage({ label = "Settings" })

	local weatherList = {}
	for weather, id in pairs(tes3.weather) do
		weatherList[#weatherList+1] = {
			label = weather,
			value = id,
		}
	end

	local inspect = require("inspect")
	local var = mwse.mcm.createVariable{
		get = function(self)
			mwse.log("get called!")
			return tes3.getCurrentWeather().index or 0
		end,
		set = function(self, newVal)
			mwse.log("set called!")
			tes3.getWorldController().weatherController:switchImmediate(newVal)
		end
	}
	page:createDropdown{
		label = "Change the Weather",
		options = weatherList,
		inGameOnly = true,
		variable = var
	}
end

event.register(tes3.event.modConfigReady, registerModConfig)
```

**Comment by C3pa** [comment]
Let's wait until #440 is merged.

**Review by jhaakma — APPROVED** [review]

**Pull request merged**
